commands/watch.go

```diff
@@ -284,12 +285,10 @@ func (c *LotusChainNotifier) Run(ctx context.Context) error {
 	for {
 		select {
 		case <-ctx.Done():
-			c.Cancel(ctx.Err())
-			return nil
+			return ctx.Err()
 		case headEvents, ok := <-hc:
 			if !ok {
-				c.Cancel(xerrors.Errorf("ChainNotify channel closed"))
-				return nil
+				return xerrors.Errorf("ChainNotify channel closed")
```

Comment:
I believe these were the only usages of the `Cancel` method (line 255). Can we nix it if it's no longer being used?

Reply:
Yeah maybe. I still think we may want to use it, but let's see how it turns out.